### PR TITLE
Add Haskell, Java, Scala

### PR DESCRIPTION
**Pull request by relrod**
The taglines probably need some work though. ;)

### CODE REVIEW

File: questions/includes/fedora/coding.yml [hunk 1]
```diff
@@ -8,8 +8,24 @@ tree:
...
     - title: C
       subtitle: close to the metal
-      href: http://lkml.org
+      href: https://lkml.org
+
+    - title: Haskell
+      subtitle: a nondynamic language for rapid development
+      segue1: Yay, types!
+      segue2: How about...
+      href: https://fedoraproject.org/wiki/Haskell_SIG
```
Comment (ralphbean) [inline questions/includes/fedora/coding.yml]: FYI, the `href` has no effect if there are `children` present.  The `href` is only for leaf nodes.

Comment (ralphbean) [inline questions/includes/fedora/coding.yml]: So, I'd move the SIG link into another child node in `includes/fedora/coding/haskell.yml`.

**Comment by ralphbean** [comment]
> The taglines probably need some work though. ;)

They look good to me, but we can improve them over time too.

I'll merge if you can fix my one comment about the `href`.  :)

**Comment by relrod** [comment]
> I'll merge if you can fix my one comment about the href. :)

Done!

**Comment by ralphbean** [comment]
Super great.  Thanks!  :)

**Pull request merged**